.github/workflows/build.yml

```diff
@@ -70,14 +71,20 @@ jobs:
         env:
           CHEF_LICENSE: "accept"
       - name: Validate Output CKL with xmllint
-        if: matrix.os == 'ubuntu-latest' || matrix.os == 'ubuntu-16.04'
+        if: matrix.platform == 'ubuntu-latest' || matrix.platform == 'ubuntu-16.04'
         uses: ChristophWurst/xmllint-action@v1
         with:
           xml-file: inspec2ckl_test_1.ckl
           xml-schema-file: test/schemas/U_Checklist_Schema_V2-3.xsd
       - name: Validate Output CKL with xmllint
-        if: matrix.os == 'ubuntu-latest' || matrix.os == 'ubuntu-16.04'
+        if: matrix.platform == 'ubuntu-latest' || matrix.platform == 'ubuntu-16.04'
         uses: ChristophWurst/xmllint-action@v1
         with:
-          xml-file: inspec2ckl_test_2.ckl
+          xml-file: test/data/inspec2ckl_test_2.ckl
           xml-schema-file: test/schemas/U_Checklist_Schema_V2-3.xsd
+      - name: Validate XCCDF 1.1 with xmllint
+        if: matrix.platform == 'ubuntu-latest' || matrix.platform == 'ubuntu-16.04' 
```

Comment:
Thanks. Any objection to adding a .yamllint file to the project to catch this and some other things in this file?

See https://yamllint.readthedocs.io/en/stable/index.html

Reply:
I'm not sure because in order for the linter to actually run the YAML would have to be correct since the linting would presumably occur within GH actions